metadata_test.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"testing"
+)
+
+func TestGithubRegexp(t *testing.T) {
+	cases := map[string]string{
+		`git https://github.com/paultag/go-config">`: "paultag/go-config",
+		`https://github.com/owner/repo->`:            "owner/repo-",
```

Comment:
did you mean `https://github.com/owner/repo->`? Actually I haven't seen someone using `-` in his repo name. But it's legal for GitHub, just created a test repo https://github.com/zhsj/test-